test/spec/spec.controller.js

```diff
@@ -20,6 +20,11 @@ describe('lib/base-controller', () => {
     sinon.stub(BaseController.prototype, 'locals').returns({foo: 'bar'});
   });
 
+  it('sets the correct error class to the instance', () => {
+    controller = new Controller({});
+    controller.Error.should.be.equal(require('../../lib/error'));
```

Comment:
Sure, any reason? I used to always put them at the top, but on a different PR you included one mid-file. Think your reasoning was it was only being used once so no point assigning it to a var (I may be wrong)